UI/src/view/linear_asset/linearAssetLayer.js

```diff
@@ -199,10 +199,10 @@ root.LinearAssetLayer  = function(params) {
   var verifyClickEvent = function(properties, evt){
     var singleLinkSelect = evt.mapBrowserEvent.type === 'dblclick';
     selectedLinearAsset.open(properties, singleLinkSelect);
-    highlightMultipleLinearAssetFeatures();
+    me.highlightMultipleLinearAssetFeatures(selectToolControl);
```

Comment:
Isn't correct to create a new selectToolControl for the care class layer, instead of passing it as an argument in here?